notebooks/official/model_evaluation/custom_tabular_classification_model_evaluation.ipynb

```diff
@@ -0,0 +1,1636 @@
+{
```

Comment:
<p>Why not just use the scikit-learn prebuilt container?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/1219/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='58'/>

Reply:
Tried it but was unable to properly load the sklearn pipeline (preprocessing + estimator) file using the pre-built image. Moreover, the serving application includes the code(postprocessing) to format the predictions as needed.